src/ChainSafe.GamingSDK.EVM.Tests/ProvidersSendTests.cs

```diff
@@ -1,59 +1,72 @@
-﻿using System;
-using System.Threading.Tasks;
-using ChainSafe.GamingSDK.EVM.Web3.Core;
-using ChainSafe.GamingSDK.EVM.Web3.Core.Evm;
+﻿using System.Diagnostics;
+using System.Linq;
+using ChainSafe.GamingSDK.EVM.Tests.Node;
 using ChainSafe.GamingWeb3;
-using ChainSafe.GamingWeb3.Build;
-using Microsoft.Extensions.DependencyInjection;
 using Nethereum.Hex.HexTypes;
 using NUnit.Framework;
-using Web3Unity.Scripts.Library.Ethers.JsonRpc;
-using Web3Unity.Scripts.Library.Ethers.NetCore;
 using Web3Unity.Scripts.Library.Ethers.Providers;
-using Web3Unity.Scripts.Library.Ethers.Signers;
 using Web3Unity.Scripts.Library.Ethers.Transactions;
+using static ChainSafe.GamingSDK.EVM.Tests.Web3Util;
 
 namespace ChainSafe.GamingSDK.EVM.Tests
 {
+    using static RpcProviderExtensions;
     using Web3 = ChainSafe.GamingWeb3.Web3;
 
     [TestFixture]
     public class ProvidersSendTests
     {
-        private Web3 web3;
+        private Web3 firstAccount;
         private Web3 secondAccount;
+        private Process node;
 
         [OneTimeSetUp]
         public void SetUp()
         {
-            web3 = Web3Util.CreateWeb3().Result;
-            secondAccount = Web3Util.CreateWeb3(1).Result;
+            node = Emulator.CreateInstance();
+
+            var firstAccountTask = CreateWeb3(0).AsTask();
+            firstAccountTask.Wait();
```

Comment:
Why not just request "Result" right ahead instead of "Wait()" here?

Reply:
We could also make this method (as well as all the test methods) async and use await then.

Reply:
This is restriction of NUnit, unfortunately we can't do `OneTimeSetUp` and `OneTimeTearDown` `asnyc`.

Reply:
Ah and overall it is a bad idea to request `Result` out of `ValueTask` since execution is not guarantied.

https://devblogs.microsoft.com/dotnet/understanding-the-whys-whats-and-whens-of-valuetask/
```
The short rule is this: with a ValueTask or a ValueTask<TResult>, you should either await it directly (optionally with .ConfigureAwait(false)) or call AsTask() on it directly, and then never use it again, e.g.
```